static/sass/_blockquote.scss

```diff
@@ -1,7 +1,7 @@
 // These are the blockquote rules from the Vanilla Dev branch at commit 19a80ec
 // https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_base_blockquotes.scss
 // When it lands in main and the depency is updated in this repo this can be removed
-@import 'vanilla-framework/scss/settings';
+@import "vanilla-framework/scss/settings";
```

Comment:
Judging from the comment on top of this file needed at all?
The commit it mentions is very old, and the blockquote styles in Vanilla look almost identical (apart from some small spacing differences). It should use the official Vanilla styles ideally.

Reply:
Good point, removed both the files